arm-servermanagement/2016-07-01-preview/swagger/servermanagement.json

```diff
@@ -1580,6 +1581,16 @@
               "type": "string",
               "format": "date-time",
               "description": "the date/time of the last published gateway"
+            },
+            "installerDownload": {
+              "type": "string",
+              "readOnly": true,
```

Comment:
is this change okay to merge now, I can revert the read-only if its not recommended